modules/cstrike/cstrike/CstrikeUtils.cpp

```diff
@@ -47,3 +48,22 @@ bool UTIL_CheckForPublic(const char *publicname)
 
 	return false;
 }
+
+void UTIL_StringToLower(const char *str, char *buffer, size_t maxlength)
```

Comment:
Well, others occurrences use a simplified version directly integrated to code. We don't have yet some manager with all those util functions so it can be used in modules. Could be done later.